src/kedro_templar/process_templates.py

```diff
@@ -116,9 +100,9 @@ def upload_file_to_s3(file_name, bucket, object_name=None):
     type=bool
 )
 @click.option(
-    "-s",
-    "--source",
-    "config_source",
```

Comment:
Why did you removed this part? This command should remains as it is. Uploading and downloading definition.yaml should be handled as a separate command. See comments in the jira task.

Reply:
it was a flag to upload config, but i changed way of uploading this